code/modules/clothing/clothing.dm

```diff
@@ -331,7 +331,7 @@ BLIND     // can't see anything
 /obj/item/clothing/under/New()
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
-		sensor_mode = pickweight(list(0 = 1, 1 = 2, 2 = 3, 3 = 2))
+		sensor_mode = pick(0, 1, 1, 2, 2, 2, 3, 3))

```

Comment:
unimportant but for the record you could also format this as 

pick(1;0, 2;1, 3;2, 2;3)